@xen-orchestra/lite/src/stories/web-core/status/vts-status.story.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <ComponentStory
+    v-slot="{ properties }"
+    :params="[
+      prop('status')
+        .required()
+        .enum(...statuses)
+        .type('Status')
+        .preset(true)
+        .widget(),
```

Comment:
The prop `iconOnly` is missing in the story.
Also, maybe it will be nice to have the full info for the `Status` type in `vts-status.story.md`